main.py

```diff
@@ -224,10 +216,139 @@ def callback(ch, method, properties, body, ctx):
 
     events = Events(ctx.config.app_cfg["rabbitmq"]["outgoing"], ctx)
     events.publish(json.dumps(param_dict), properties.correlation_id)
+    return True
+
 
+def delete_media_object(mediahaven_service, fragment_id: str, reason: str) -> bool:
```

Comment:
Should better be a method on the MediaHaven-svc.
And a bit awkwardly placed in between the create and te remove handler.

Reply:
The actual delete method is in MH-svc. This method logs and has some error handling.
It is placed just above the method that handles the delete events, as that one uses this logic.

Shall I move it above the create handler?